common/service/config/config.go

```diff
@@ -232,6 +232,8 @@ type (
 		Level string `yaml:"level"`
 		// OutputFile is the path to the log output file
 		OutputFile string `yaml:"outputFile"`
+		// levelKey is the desired log level
```

Comment:
how about: levelKey is the desired log level, defaults to "level"

Reply:
done